Actions/ReadSettings/ReadSettings.ps1

```diff
@@ -18,12 +18,11 @@ Param(
 )
 
 $telemetryScope = $null
-$bcContainerHelperPath = $null
 
 try {
     $baseFolder = $ENV:GITHUB_WORKSPACE
     . (Join-Path -Path $PSScriptRoot -ChildPath "..\AL-Go-Helper.ps1" -Resolve)
-    $BcContainerHelperPath = DownloadAndImportBcContainerHelper -baseFolder $baseFolder
+    DownloadAndImportBcContainerHelper -baseFolder $baseFolder
```

Comment:
Why is _BCContainerHelper_ needed for reading the settings?

Reply:
The telemetry pieces are built on functionality in containerHelper (nesting all telemetry within a workflow).
We might want to refactor the telemetry at some point in time to not use containerhelper, but that is another story.